Story/Voltaire/CoreVoltaire.cs

```diff
@@ -0,0 +1,432 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class CoreVoltaire
+{
+    public IScriptInterface Bot => IScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreStory Story = new();
+
+    public void ScriptMain(IScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        Core.RunCore();
+
+        Core.SetOptions(false);
+    }
+
+    public void DoAll()
+    {
+        if (!Core.IsMember)
+        {
+            Core.Logger("Voltaire Saga Is Member Only. Skipping this Script");
+            return;
+        }
+
+        FiveSaloon();
+        Train();
+        Blackstone();
+        FireTown();
+        FireRiver();
+        FireTunnel();
+        Sleezter();
+        Neverlore();
+        Skullpunch();
+        Vampirates();
+        TreasureIsland();
+        GustavduGrog();
+        Wormhole();
+        Crownsreach();
+    }
+
+    public void FiveSaloon()
+    {
+            
+        if (Core.isCompletedBefore(1049))
+            return;
+
+        //Unknown name 1042
+        Story.ChainQuest(1042);
+
+        //Exit, Stage Right 1043
+        Story.MapItemQuest(1043, "fivesaloon", 406, 1);
+
+        //The Wedge 1044
+        Story.KillQuest(1044, "fivesaloon", "Bulletless Bandit");
+
+        //Freedom? 1045
+        Story.MapItemQuest(1045, "fivesaloon", 407, 1);
+
+        //Other Ideas?! 1046
+        Story.KillQuest(1046, "fivesaloon", "Storagebox");
+
+        //Got a Light 1047
+        Story.KillQuest(1047, "fivesaloon", "Bulletless Bandit");
+
+        //BADABOOM! 1048
+        Story.MapItemQuest(1048, "fivesaloon", 408, 1);
+
+        //Lack of Subtlety 1049
+        Story.KillQuest(1049, "fivesaloon", "One-Armed Bandit");
+    }
+
+     public void Train()
+    {
+        if (Core.isCompletedBefore(1053))
+            return;
+
+        //Look Around 1050
+        Story.MapItemQuest(1050, "train", 409, 1);
+
+        //Caboose Key 1051
+        Story.KillQuest(1051, "train", "Ghostly Conductor");
+        
+        //Slow your Role 1052
+        Story.MapItemQuest(1052, "train", 410, 1);
+        
+        //Slow the Coal 1053
+        Story.KillQuest(1053, "train", "Coal Ghoul");
+    }
+
+     public void Blackstone()
```

Comment:
i remember u told me that blackstone depends on Train map to be unlocked if so add 
Train();
in the first line of black stone
see moglooween and candyshop for an example